src/templates/resources/footer.json

```diff
@@ -1231,6 +1229,14 @@
                 "url": "https://connect.menulog.co.nz/login?client_id=RappsOrigPartnerCentre&response_type=code&redirect_uri=https:%2F%2Fpartner.menulog.co.nz%2Fsignin-jeconnect&state=QaF4ae5nfj7FUpuoevTVCHvmMBxIA1hO_4mDtePNEJOasCly6wdTLaHjTuvS7_If4SwPYZelgwT0ZFYb7CCZvfIIr9X0OVLKiRpdCLYU9zJPuorMgnFjjRlyQNdU8L_5ZvkfGGBtXJx-jNOlh5lU_aqiQ-KgrKthGa83ZbYH8GjV81bwGb7iJ8PRFJPYbwTyys-zclUwNFxXhfdsOpqIWsIlfpQ",
                 "text": "Partner Centre"
             },
+            {
+                "url": "https://couriers.menulog.co.nz/application",
```

Comment:
Should these urls work?

Reply:
- `/login` was working for me yesterday.
- `/application` seemed to be a redirect back to our `just-eat.com` domain.

Reply:
Weird, doesn't work for me at all.